src/BreweryScrapper.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Interns2022B;
+
+class BreweryScrapper
+{
+    public const SHOW = "show";
+    public const LIST = "list";
+    public const CLEAR = "clear";
+    public const EXIT = "exit";
+
+    public array $data = [];
+
+    public function dataToArray(string $name): array
```

Comment:
I would like to rename method to `getBreweries` or something similar.

Reply:
changed.